test/controllers/api/v1/stats/promo_campaigns_controller_test.rb

```diff
@@ -2,26 +2,20 @@
 
 class Api::V1::Stats::PromoCampaignsControllerTest < ActionDispatch::IntegrationTest
   test "/api/v1/stats/promo_campaigns/ returns json representation of all campaigns" do
-    promo_campaign = promo_campaigns(:test_promo_campaign)
-    promo_campaign2 = promo_campaigns(:test_promo_campaign2)
-    promo_registration = promo_registrations(:owner_promo_registration)
-    promo_registration2 = promo_registrations(:unattached_promo_registration)
     get "/api/v1/stats/promo_campaigns/", headers: { "HTTP_AUTHORIZATION" => "Token token=fake_api_auth_token" }
       assert_equal(
-        [{
-          promo_campaign_id: promo_campaign.id,
-          name: "test campaign",
-          promo_registrations: [
-            { promo_registration_id: promo_registration.id, referral_code: "PRO789" }
-          ]
+        PromoCampaign.all.map { |promo_campaign|
```

Comment:
We should change this to be a do/end block. I imagine the linter should have picked this up?

```suggestion
        PromoCampaign.all.map do |promo_campaign|
```